### PR TITLE
Fix/e2e tests

### PR DESCRIPTION
**Pull request by mzedel**
- this needs mendersoftware/mender-test-containers#178

### CODE REVIEW

File: src/js/components/settings/settings.js [hunk 1]
```diff
@@ -84,7 +84,7 @@ export const Settings = ({
         }
       });
     } else {
-      const notStripePromise = {};
+      const notStripePromise = new Promise(resolve => setTimeout(resolve, 700));
```
Comment (oleorhagen) [inline src/js/components/settings/settings.js:87]: from nothing to `700` -_-

File: tests/e2e_tests/integration/00-setup.spec.ts [hunk 2]
```diff
@@ -87,6 +87,12 @@ test.describe('Test setup', () => {
       ]);
       const page = await context.newPage();
       await page.goto(`${baseUrl}ui/#/settings`);
+      const isVisible = await page.isVisible(`text=/Change email/i`);
+      if (!isVisible) {
+        console.log('settings may not be loaded - move around');
+        await page.goto(`${baseUrl}ui/#/help`);
```
Comment (oleorhagen) [inline tests/e2e_tests/integration/00-setup.spec.ts:93]: Q: Why `help`, then `settings`?

File: .gitlab-ci.yml [hunk 3]
```diff
@@ -204,17 +203,17 @@ test:prep:
 test:staging-deployment:chrome:
   extends: .template:test:staging-deployment
   script:
-    - npm run test
+    - npm run test-ci
```
Comment (oleorhagen) [inline .gitlab-ci.yml:206]: :raised_hands:

**Review by oleorhagen — APPROVED** [review]

File: src/js/components/settings/settings.js (hunk 1 above)
Comment (mzedel) [reply src/js/components/settings/settings.js:87]: just anything that's not nothing - usually the stripe request should be faster (they are in a `Promise.race` 🏎️  after this) and at these times it still is considered to not be perceived by a user 👍 (although 700 is on the upper end of what we use in the codebase)

File: tests/e2e_tests/integration/00-setup.spec.ts (hunk 2 above)
Comment (tranchitella) [reply tests/e2e_tests/integration/00-setup.spec.ts:93]: I suppose to trigger a refetch.

**Review by tranchitella — APPROVED** [review]
Looks good.

File: tests/e2e_tests/integration/00-setup.spec.ts (hunk 2 above)
Comment (oleorhagen) [reply tests/e2e_tests/integration/00-setup.spec.ts:93]: So the `help` part could've been pretty much anything?

Comment (mzedel) [reply tests/e2e_tests/integration/00-setup.spec.ts:93]: the help pages are the most static area of the ui so I think no additional requests to the backend, navigating there is basically a UI only thing... afterwards the step to the settings gives the stripe request another chance to complete and show the settings - this should be just the fallback for the 700 timeout from above 🤞

**Pull request merged**